NitroxModel/Packets/PlayerMovement.cs

```diff
@@ -7,18 +8,16 @@ namespace NitroxModel.Packets
     [Serializable]
     public class PlayerMovement : Movement
     {
-        public override ushort PlayerId { get; }
+        public override NitroxId Id { get; }
```

Comment:
Just an idea:
Would it be possible to send the ushort PlayerId and provide the override property with an lookup of playerID ushort to NitroxId? As we are sending a lot of these packet size reduction is a point to consider.

Reply:
I'd have to figure out a way to convert playerId to NitroxId, and have that in NitroxModel, plus add [IgnoredMember] from BinaryPack to the Id property.

The Velocity property which was removed was 12 Bytes(only counting the size of the floats), the size of the PlayerId property was 2 bytes, for a total of 14 bytes. the added size from Guid should be 16 bytes, thus a net increase of 2 bytes...